src/library_fs.js

```diff
@@ -12,6 +12,8 @@ mergeInto(LibraryManager.library, {
     '$intArrayFromString',
     '$stringToUTF8Array',
     '$lengthBytesUTF8',
+    '$mmapAlloc',
+    'emscripten_builtin_memalign',
```

Comment:
Why are these added? (I see the first in this file, so it might be valid, but I don't seem to see the second.)

Reply:
When I tried running `test_fs_js_api`, I got the error that ```RuntimeError: Aborted(internal error: mmapAlloc called but `emscripten_builtin_memalign` native symbol not exported)```. Could `emscripten_builtin_memalign` be a dependency for `mmapAlloc` that wasn't exported before?

Reply:
How are you running `test_fs_js_api`? I mean, as `wasmfs.test_fs_js_api`, or just normally. And do you mean with this PR applied?

This confuses me because this file should not be used when WasmFS is enabled, so I'm not sure how you are seeing that error.

Reply:
Normally I run both `test_fs_js_api` then `wasmfs.test_fs_js_api` to make sure the Wasm JS API is compatible, and I see the error on the legacy API (`test_fs_js_api`).

Reply:
>  I see the error on the legacy API

That's odd, since this PR shouldn't change the behavior of the legacy API, unless I'm missing something? What change in this PR affects it?

Reply:
One possibility I was considering was that `FS.mmap` was very rarely used, so when I called it in the test there was a missing import. However, I didn't change anything in the legacy `library_fs.js` besides the imports, so I'm a little stuck on where else the error could have come from.

Reply:
I was searching for `emscripten_builtin_memalign` and found these lines
https://github.com/emscripten-core/emscripten/blob/9de0509aa7bbbfe7b5d2ceb0d4c70cf35c8a7ada/src/library_syscall.js#L128-L134
It looks like `emscripten_builtin_memalign` should have been exported but I can't access it from `library_fs.js`. I'm don't fully understand what the comment means, but could the comment be related to the current issue seen when calling `FS.mmap` in the testing file?

Reply:
> I'm a little stuck on where else the error could have come from.

To debug that, I would "bisect" on the changes in this PR. That is, the non-WasmFS test should pass before this PR (since it passes on our CI). Then add some part of this PR and see if it fails, until you find the specific change in this PR that causes it. When we know the specific line that explains the problem, it might be obvious what the issue is there.

> I'm don't fully understand what the comment means

Hmm, I don't either :smile: If what I wrote above doesn't help, I can debug that (or we can wait for @sbc100 who will probably know offhand).